### DISCUSSION TITLE
Header and Footer not properly rendered in default.vue Nuxt3

### DISCUSSION THREAD

**Discussion by ErinTran15** [nuxt/nuxt#23757] [Questions]
I'm trying to render a Header and a Footer in default.vue 
```
<template>
  <div>
    <Header />
    <slot />
    <Footer />
  </div>
</template>
```

At times, the Header and Footer components appear, while other times they do not. Upon inspecting the Vue DevTools, I've noticed that they are indeed rendered, but their height is consistently 0. However, when they do appear, their height is typically around 93 or some other non-zero value. I'm perplexed as to why they're exhibiting this behavior. I've attempted various solutions, such as directly inserting the code for Header and Footer into the default.vue file, but the issue persists. does anybody had this issue before or can help me out with this?

**Comment by warflash** [comment]
Can you provide a stackblitz to investigate? https://nuxt.new/s/v3
